eth/catalyst/api.go

```diff
@@ -410,13 +410,20 @@ func (api *ConsensusAPI) getPayload(payloadID engine.PayloadID) (*engine.Executi
 // NewPayloadV1 creates an Eth1 block, inserts it in the chain, and returns the status of the chain.
 func (api *ConsensusAPI) NewPayloadV1(params engine.ExecutableData) (engine.PayloadStatusV1, error) {
 	if params.Withdrawals != nil {
-		return engine.PayloadStatusV1{Status: engine.INVALID}, fmt.Errorf("withdrawals not supported in V1")
+		return engine.PayloadStatusV1{Status: engine.INVALID}, engine.InvalidParams.With(fmt.Errorf("withdrawals not supported in V1"))
```

Comment:
Tested this, seems that the error that the remote "sees" is `can't execute payload: Invalid parameters`"

Reply:
More specifically: 
```
{"jsonrpc":"2.0","id":null,"error":{"code":-32600,"message":"invalid request"}}
```

Reply:
To be specific

```
{"jsonrpc":"2.0","id":1,"error":{"code":-32602,"message":"Invalid parameters","data":{"err":"withdrawals not supported in V1"}}}

```
LGTM!